static/app/components/forms/controls/selectControl.tsx

```diff
@@ -20,6 +15,12 @@ import styled from '@emotion/styled';
 import omit from 'lodash/omit';
 
 import {Chevron} from 'sentry/components/chevron';
+import {
+  createFilter,
+  mergeStyles,
+  ReactSelect,
+  selectComponents,
+} from 'sentry/components/forms/controls/reactSelectWrapper';
```

Comment:
the wrapper is good trick 👍 . Maybe we should accommodate it with an update to the `restrictedImportPaths` in our eslint config to avoid accidental direct imports of `react-select`:

https://github.com/getsentry/sentry/blob/48f06ee1a866b008b387f9dd4e5574ef278efccf/eslint.config.mjs#L45